.github/workflows/cd-pipeline.yml

```diff
@@ -89,14 +89,21 @@ jobs:
           
       - name: Merge commit SHA to release
         run: |
-          git merge ${{ github.sha }} -X theirs
+          git merge ${{ github.sha }} --no-squash -X theirs
```

Comment:
My understanding is that this is in lieu of reverting the previous merge.

Reply:
We did not revert the merge.  We are just readding this flag here.

Reply:
That is correct